katas/content/nonlocal_games/ghz_classical_game/index.md

```diff
@@ -0,0 +1,8 @@
+**Inputs:**
+
+1. An operations which implement classical strategies (i.e., takes an input bit and produces an output bit),
+2. An array of 3 input bits that should be passed to the players.
+
+**Goal:**
```

Comment:
I try to use "goal" for operations that have no return value (and act via side effects on qubits) and "output" for operations that actually return something
```suggestion
**Output:**
```